app/assets/stylesheets/components/_homepage.scss

```diff
@@ -44,3 +44,21 @@
   color: #2B3D29;
 
 }
+
+.spinner-container {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-top: 100px;
+}
+
+.fas {
+  // display: block;
+  color: $dark-green;
+  font-size: 24px;
+}
```

Comment:
@MarlAndre May i suggest that we use nested CSS properties here? Because the `.fas` will apply to all other icons with the same class. To avoid that, you may move lines 60-64 inside the `.spinner-container`, so the `.fas` class here is only for this spinner :)

e.g.

```
. spinner-container {
  position: absolute
  ...
  .fas {
    color: $dark-green;
    ...
  } 
}
```

Reply:
Will do. Thanks !